backend/benefit/common/tests/conftest.py

```diff
@@ -1,10 +1,27 @@
 import factory.random
 import pytest
-from django.test.utils import freeze_time
+from django.contrib.auth.models import Permission
+from freezegun import freeze_time
+from rest_framework.test import APIClient
+from users.tests.factories import UserFactory
 
 
 @pytest.fixture(autouse=True)
 def setup_test_environment(settings):
     factory.random.reseed_random("777")
     with freeze_time("2021-01-04"):
         yield
+
+
+@pytest.fixture
```

Comment:
Good to have these in the common conftest, I have identical in applications/tests which can be removed once these are merged